src/pages/search/advanced_search/FileStatusSelect.tsx

```diff
@@ -0,0 +1,50 @@
+import { FormControl, InputLabel, MenuItem, Select, SelectChangeEvent } from '@mui/material';
+import { useTranslation } from 'react-i18next';
+import { useHistory } from 'react-router-dom';
+import { ResultParam } from '../../../api/searchApi';
+import { dataTestId } from '../../../utils/dataTestIds';
+
+enum FileStatus {
+  hasPublicFiles = 'hasPublicFiles',
+  noFiles = 'noFiles',
+}
+
+export const FileStatusSelect = () => {
+  const { t } = useTranslation();
+  const history = useHistory();
+  const searchParams = new URLSearchParams(history.location.search);
+  const selectedParam = searchParams.get(ResultParam.Files) ?? '';
+
+  const handleChange = (event: SelectChangeEvent) => {
+    const newValue = event.target.value;
+
+    if (newValue !== '') {
+      searchParams.set(ResultParam.Files, newValue);
+      history.push({ search: searchParams.toString() });
+    } else {
+      searchParams.delete(ResultParam.Files);
+      searchParams.delete(ResultParam.From);
+      history.push({ search: searchParams.toString() });
+    }
+  };
+
+  return (
+    <FormControl sx={{ minWidth: '10rem' }} size="small">
+      <InputLabel id="file-status-select-label">{t('registration.files_and_license.files')}</InputLabel>
+      <Select
+        data-testid={dataTestId.startPage.advancedSearch.fileStatusSelect}
+        labelId="file-status-select-label"
+        value={selectedParam}
+        label={t('registration.files_and_license.files')}
+        onChange={handleChange}>
+        <MenuItem value="">
+          <em>{t('common.select')}</em>
```

Comment:
`<em>` er for å legge ekstra trykk på noe (emphasize), virker som det som er ønsket her er at det bare skal være italic?
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/em